openreview/stages/venue_stages.py

```diff
@@ -1155,6 +1163,39 @@ def get_mandatory_readers(self, conference, number):
         if conference.use_senior_area_chairs:
             readers.append(conference.get_senior_area_chairs_id(number))
         return readers
+    
+    def get_description(self, conference):
+
+        if self.description:
+            return self.description
+        
+        instructions = '''
+- Please provide your official comments in the space provided below.
```

Comment:
This is what I think the description should say (1 and 4 are always present, only one from 2 and 3):

1.Please select who should be able to see your comment under “Readers”. Readers marked as mandatory are required in order to post the comment. 

---------------------
2a. Program Chairs will be notified of all comments.
2b. Program Chairs will be notified of any comments that are visible only to the mandatory readers.
2c. Program Chairs will not be notified of any of your comments.

---------------------
3a. Senior Area Chairs will be notified of all comments.
3b. Senior Area Chairs will be notified of any comments that are visible only to the mandatory readers.

---------------------
4.All other readers will be notified of all comments.